packages/atlas/src/components/_nft/NftWidget/NftHistory.styles.ts

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled'
+
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { cVar, sizes } from '@/styles'
+
+import { SizeProps, sizeObj } from './NftWidget.styles'
+
+type OpenProps = { 'data-open': boolean }
+
+export const NftHistoryHeader = styled.div<SizeProps & OpenProps>`
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  padding: ${sizes(6)};
+  user-select: none;
+  cursor: pointer;
+
+  &[data-size=${sizeObj.small}] {
+    padding: ${sizes(4)};
+  }
+
+  &[data-open='true'] {
+    padding-bottom: 0;
+  }
+`
+
+export const StyledChevronButton = styled(Button)<OpenProps>`
+  transform: rotate(0);
+  transform-origin: center;
+  transition: ${cVar('animationTransitionFast')};
+
+  &[data-open='true'] {
+    transform: rotate(180deg);
+  }
+`
+
+export const FadingBlock = styled.div<{ width: number } & { 'data-bottom'?: boolean } & SizeProps>`
```

Comment:
Why not this way? Also maybe we could move declare it above? Did you intentionally not use `OpenProps`?
```suggestion
export const FadingBlock = styled.div<{ width: number; 'data-bottom'?: boolean } & SizeProps>`
```

Reply:
yes, open props was not needed in this component